packages/db/src/schema.ts

```diff
@@ -104,6 +104,115 @@ export const sessions = authSchema.table(
   ],
 );
 
+/**
+ * CLI sessions for device authentication
+ *
+ * Separate from auth.sessions (managed by NextAuth) to maintain clean separation.
+ * References the browser session that authorized it, but independent lifecycle.
+ *
+ * Token stored as `id` is a 256-bit high-entropy credential - NEVER log it.
+ */
+export const cliSessions = pgTable(
+  "cli_sessions",
+  ({ text, timestamp, uuid }) => ({
+    // High-entropy session token (256 bits, base64url encoded)
+    // This IS the credential - treat as secret
+    id: text("id").primaryKey().notNull(),
+    // The user who owns this CLI session
+    userId: uuid("user_id")
+      .references(() => authUsers.id, { onDelete: "cascade" })
+      .notNull(),
+    // The browser session that authorized this CLI session (for audit/reference)
+    // Set to null if browser session is deleted - CLI session survives
+    browserSessionId: text("browser_session_id").references(() => sessions.id, {
+      onDelete: "set null",
+    }),
+    // When this CLI session expires (required, typically 90 days)
+    notAfter: timestamp("not_after", { withTimezone: true }).notNull(),
+    createdAt: timestamp("created_at", { withTimezone: true })
+      .defaultNow()
+      .notNull(),
+    updatedAt: timestamp("updated_at", { withTimezone: true })
+      .defaultNow()
+      .notNull(),
+  }),
+  (table) => [
+    index("cli_sessions_user_id_idx").on(table.userId),
+    index("cli_sessions_not_after_idx").on(table.notAfter),
+  ],
+);
+
+export type DBCliSession = typeof cliSessions.$inferSelect;
+
+/**
+ * Device auth codes for CLI authentication (OAuth 2.0 Device Authorization Grant)
+ * These are temporary codes that expire after 15 minutes.
+ * Once a code is verified, a CLI session is created in cli_sessions.
+ */
+export const deviceAuthCodes = pgTable(
+  "device_auth_codes",
+  ({ text, timestamp, uuid, boolean }) => ({
+    id: text("id")
+      .primaryKey()
+      .notNull()
+      .unique()
+      .default(sql`generate_custom_id('dac_')`),
+    // Server-generated UUID used by CLI to poll for completion
+    deviceCode: text("device_code").notNull().unique(),
```

Comment:
If this is a server-generated UUID, then use the `uuid()` data type for the column and use `gen_random_uuid()` as the default